cleancat/base.py

```diff
@@ -1,13 +1,12 @@
 import datetime
 import re
-import sys
 
 import pytz
+import six
```

Comment:
I prefer to introduce as little dependencies as possible, so if `six` is only required for this one line I wouldn't do it (when I wrote this code  I was aware that `six` contains this but specifically decided against adding a dependency).

Next one I want to get rid of is dateutil...

Reply:
That's fair. I saw that `six` was already a dependency of CleanCat via `python-dateutil` and thought we might as well use it... But since we want to get rid of dateutil, it makes sense to not rely on `six` either. Will fix.

Reply:
Fixed in eec52c6